example/tile.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+const LifxClient = require('../lib/lifx').Client;
+
+const client = new LifxClient();
+
+const LOOPTIME = 1000;
+const TILE_LABEL = process.argv.length > 2 ? process.argv[process.argv.length - 1] : '*';
```

Comment:
Do we want this to come from the command line execution or could it be done differently? Also, there's a library `yargs` which makes it a lot easier to pull in command line parameters. If we're doing this in Node.js anyway, why not pull it in from a config when creating the LIFX client?

Reply:
Agree on this one. At least the script should fail if run without arguments and instruct on the correct usage. Example:
```
Usage: node tile.js <label>, e.g. node tile.js "My Tiles"
```